ros2_control_demo_bringup/config/rrbot_controllers.yaml

```diff
@@ -8,7 +8,7 @@ controller_manager:
     forward_position_controller:
       type: forward_command_controller/ForwardCommandController
 
-    position_trajectory_controller:
+    joint_trajectory_controller:
```

Comment:
What if we call it "joint_trajectory_position_controllers"?

There are multiple versions of trajectory controller, depending on which interfaces they are controlling.

Reply:
I think that joint_trajectory_position_controller is a perfect fit to the intent behind the example and the name of the controller+interface usage 

I will make the necessary changes on Monday